libass/ass_priv.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2006 Evgeniy Stepanov <eugeni.stepanov@gmail.com>
```

Comment:
You’re not wrong, but I don’t know better either. I figured since this code was moved from ass.c, I’d copy the copyright notice from there, too. It’s even somewhat true, as half of `struct parser_priv` does in fact date back to eugeni’s 2006 commits (although I guess that’s a small fraction of the current code lines overall).

Reply:
We should probably switch to some kind of generic “libass contributors” copyright notice in all files. Or, you know, just drop copyright notices altogether, since they don’t have any legal significance anyway.